### DISCUSSION TITLE
possibility to see the MAC-adresses on windows 10 app/export settings to import them to android app

### DISCUSSION THREAD

**Discussion by h3rm** [emericg/WatchFlower#31] [Q&A] [answered]
Hi! i just bought the android app and want to switch from windows 10 app to android to monitor 5 flower care sensors.. is it somehow possible to see which named sensor (eg. tomato balkony 1, tomato seed greenhouse 1, etc.) has which MAC so i don't have to pull out each sensor, drop it in water to see if it's the correct one and configure it on the "new" android app? or otherwise: it is possible to export all settings from windows app and import it to the android app? thanks! :)

**Comment by emericg** [comment, marked as answer]
Hi,
You can't import data and settings unfortunately, but you can identify sensors using two ways.

The first is the little "blink led" action, that will... blink the flower care led:
![Capture d’écran de 2021-03-07 18-40-34](https://user-images.githubusercontent.com/691258/110249154-095e2c80-7f75-11eb-8e99-de1638b41721.png)

Or yes you can just read the MAC address, it's in the last submenu:
![Capture d’écran de 2021-03-07 18-40-44](https://user-images.githubusercontent.com/691258/110249155-09f6c300-7f75-11eb-9260-685e0ee65cb9.png)

**Comment by h3rm** [comment]
nice, thx! i didn't recognized that the MAC is shown in the settings, sorry! just configured all 5 sensors in 5minutes :) thanks for the fast reply and thanks for this really nice app(s) :)

**Reply by emericg** [reply]
You're welcome ^^
